windows/rngallery/MainPage.cpp

```diff
@@ -17,19 +16,5 @@ namespace winrt::rngallery::implementation
         InitializeComponent();
         auto app = Application::Current().as<App>();
         ReactRootView().ReactNativeHost(app->Host());
-        // Workaround for https://github.com/microsoft/react-native-windows/issues/6287
```

Comment:
We'll want to add this code back in. Needed for a webview workaround and versioning.